wiki/osu!wiki_contribution_guide/en.md

```diff
@@ -1,936 +1,80 @@
-As you can see, everything has been moved to [GitHub][GitHub osu-wiki].
+3. [Download and install GitHub Desktop](https://desktop.github.com) or another tool to use Git. This guide leads you through the workflow with GitHub Desktop only though.
+4. Run GitHub Desktop and sign in to GitHub.com after installation.
+5. Clone your forked repository to your local PC. For that, go to the page of your forked repository and click the green "Clone or download" button. Copy the shown URL from it and use it
 
-So here are a few things you should have in mind to get you started for contributing to the osu!wiki.
+    ![](img/clone-repository-1.png "Get the URL from your forked repository page")
 
-## Disclaimer
+    ![](img/clone-repository-2.png "Click the 'Clone a repository' button")
 
-This quick-start guide is intended for newcomers not familiar with the feature-branch workflow.
-If you have previous experience in using GitHub, feel free to skip this guide and remember to follow feature-branch workflow for content update.
+    ![](img/clone-repository-3.png "Enter the URL to the repository")
 
-For complete reference and help with using GitHub, please visit [GitHub Help][GitHub Help].
+    ![](img/clone-repository-4.png "Cloning the repository might take a while")
 
-For a styling guide when writing a page, check out [Article Style Guide][ASG] instead.
+## How to use GitHub Desktop
 
-If you have any questions, you can inquire at the [osu!dev Discord Server][osu!dev Discord], under ``#osu-wiki`` preferably.
+Before working on the osu!wiki files, make sure that the files you work on are up-to-date. Working on outdated files will lead to *conflicts* and your work might be futile.
 
-Reading time for the whole page is around an hour.
+To keep your fork updated, do the following:
 
-## Prerequisites
+1. Go to the GitHub Desktop client while you have your forked repository opened
+2. Click "Repository" on the menu header, then "Open command prompt". A black command prompt window should open.
+3. If you do this for the first time, enter `git remote add upstream https://github.com/ppy/osu-wiki.git` into the command prompt. Then enter `git fetch upstream`. This step can be skipped in future updates.
 
-To contribute to the osu!wiki, you must have the following:
+  ![](img/command-prompt-1.png "Enter the mentioned commands into the command prompt and press ENTER")
 
-- A [GitHub account][GH account].
-- Knowledge in [Markdown][MD], specifically [GitHub-Flavoured Markdown(GFM)][GFM].
-- [Understanding the Git's workflow (feature-branch workflow)][GH Help feature branch].
-  - [A simplified visual explanation provided by GitHub Guides][GH Guide feature branch].
-- Able to write in English, preferably British English for consistency.
-  - Everything not related to the page content itself (commit messages, pull requests, filenames, etc) **must** be written in English.
+4. To update your forked repository, enter `git pull upstream master` into the command prompt.
```

Comment:
`git pull upstream master` appears to be a shorthand of:
```
git fetch upstream/master
git merge upstream/master
```

Personally I'd remove the mention of `git fetch upstream` from step 3 and just tell people to fetch a specific branch whenever they want to merge it into a repo.